inference-engine/src/transformations/src/transformations/common_optimizations/compress_float_constants.cpp

```diff
@@ -99,21 +99,7 @@ ov::pass::AddOldApiMapToParameters::AddOldApiMapToParameters() {
             return false;
         auto p_type = param_node->get_element_type();
         if (p_type == ov::element::f32 || p_type == ov::element::f64) {
-            std::vector<uint64_t> order;
-            if (ov::has_old_api_map(node)) {
```

Comment:
Why you don't add order for Parameter?

Reply:
Order is not changed in this part. It sets only Parameter type and code below formed default transpose order with [0 .. rank-1] values. As attributes are separated now it is not needed anymore.